core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -46,6 +47,35 @@ public static MetricsConfig getDefault() {
     return spec;
   }
 
+  public static Map<String, String> updateProperties(Map<String, String> props, List<String> deletes,
+                                                     Map<String, String> renames) {
```

Comment:
nit: I think `deletedColumns` and `renamedColumns` would be a bit better. That's exactly the variable names we pass in from `SchemaUpdate`.